study.md

```diff
@@ -103,7 +111,9 @@ Mollusca, and find the last record.
 | 2 | Stubby Squid | Rossia pacifica | Mollusca |
 
 ```ruby
-# your answer here
+Organism.find_by(common_name: "Red Panda")
+Organism.where(phylum: 'Mollusca')
+Organism.find(-1)
```

Comment:
`#find` takes an `id` as a parameter and `id`s are always non-negative integers.  `Organism.last`.